.github/workflows/e2e_tests.yaml

```diff
@@ -55,14 +55,10 @@ jobs:
         if: steps.avd-cache.outputs.cache-hit != 'true'
         uses: reactivecircus/android-emulator-runner@50986b1464923454c95e261820bc626f38490ec0
         with:
-          api-level: 30
-          target: google_atd
-          arch: x86
-          profile: Galaxy Nexus
-          force-avd-creation: false
-          emulator-options: -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none
-          disable-animations: false
-          channel: canary
+          api-level: 33
+          arch: x86_64
+          target: google_apis
```

Comment:
Looking at documentation `google_atd` was supposed to make tests go faster.
Do you think we should try `google_atd` with api-level 33?

https://github.com/marketplace/actions/android-emulator-runner#configurations

Reply:
If it works with google_apis and is more stable it might not be worth it tho

Reply:
@russellwheatley I think you need to add canary:
```
Note that aosp_atd and google_atd currently require the following: api-level: 30, arch: x86 or arch: arm64-v8 and channel: canary.
```

Reply:
Emulator fails to start with the `google_atd` setup, reverted back to what it was previously.